app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -391,11 +391,11 @@
   <dimen name="solution_summary_margin_top_linear_layout">0dp</dimen>
   <dimen name="solution_summary_margin_bottom_linear_layout">32dp</dimen>
   <dimen name="solution_summary_margin_top_solution_summary_label">16dp</dimen>
-
  <!-- SectionTitleFragment -->
   <dimen name="section_title_divider_view_layout_margin_top">48dp</dimen>
   <dimen name="section_title_text_view_layout_margin_start">120dp</dimen>
   <dimen name="section_title_text_view_layout_margin_top">32dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">120dp</dimen>
-
+  <!-- StoryFragment -->
+  <dimen name="story_fragment_padding_bottom">160dp</dimen>
 </resources>
```

Comment:
As this dimension value is same as in mobile-portrait `dimens.xml` then you can revert this change.

Reply:
ditto